apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.tsx

```diff
@@ -401,93 +414,245 @@ onDocumentReady(() => {
+                }
+              >
+                {row.time_remaining_sec !== null
+                  ? row.has_open_instance
+                    ? [
+                        <option value="+1">Add to instances with time limit</option>,
+                        <option value="-1">Subtract from instances with time limit</option>,
+                      ]
+                    : [<option value="+1">Add</option>, <option value="-1">Subtract</option>]
+                  : null}
+                <option value="set_total">Set total time limit</option>
+                <option value="set_rem">Set remaining time</option>
+                <option value="set_exact">Set exact closing time</option>
+                {row.open && row.time_remaining !== 'Open (no time limit)' ? (
+                  <option value="unlimited">Remove time limit</option>
+                ) : null}
+                {row.open && row.time_remaining !== 'Expired' ? (
+                  <option value="expire">Expire time limit</option>
+                ) : null}
+              </select>
+              <TimeLimitExplanation plusMinus={form.plus_minus} />
+            </p>
+          ) : null}
+          {showTimeLimitOptions &&
+          form.plus_minus !== 'set_exact' &&
+          form.plus_minus !== 'unlimited' &&
+          form.plus_minus !== 'expire' ? (
+            <div class="input-group mb-2">
+              <input
+                class="form-control time-limit-field"
+                type="number"
+                name="time_add"
+                aria-label="Time value"
+                value={form.time_add}
+                onChange={(e) => updateFormState('time_add', parseFloat(e.currentTarget.value))}
+              />
+              <span class="input-group-text time-limit-field">minutes</span>
+            </div>
+          ) : null}
+          {showTimeLimitOptions && form.plus_minus === 'set_exact' ? (
+            <div class="input-group date-picker mb-2">
+              <input
+                class="form-control date-picker"
+                type="datetime-local"
+                name="date"
+                value={form.date}
+                onChange={(e) => updateFormState('date', e.currentTarget.value)}
+              />
+              <span class="input-group-text date-picker">{timezone}</span>
+            </div>
+          ) : null}
+          {(row.open || !form.reopen_without_limit) &&
+          (form.plus_minus === 'set_total' ||
+            form.plus_minus === 'set_rem' ||
+            form.plus_minus === '+1' ||
+            form.plus_minus === '-1') ? (
+            <p>
+              Proposed closing time: {proposedClosingTime()} {timezone}
+            </p>
+          ) : null}
+          <p>
+            {row.has_closed_instance ? (
+              <div class="form-check">
                 <input
                   class="form-check-input"
                   type="checkbox"
                   name="reopen_closed"
-                  value="true"
```

Comment:
Why did you drop `value` here? Not that it _really_ matters, but you left it on other radio inputs, so it'd be nice to keep that consistent.

Reply:
Restored `value="true"` in [42543c9](https://github.com/PrairieLearn/PrairieLearn/pull/10800/commits/42543c9a1ca769d8b9f72a66c1f6d1916060f477).